### PR TITLE
Partner Deal Form

### PR DESCRIPTION
**Pull request by ksatx**
Creating a partner deal registration form for Tonia. Use case: a place for Axway and other partners to refer their customers to us.

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit c8dfb1db792f6c685cf22097fa2659e75f8d77c2

https://deploy-preview-1439--stoplightio.netlify.com

File: netlify/forms/partner-deal.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+path: /partner-deal-registration
+title: Stoplight Partner Deal Registration
+subtitle: >-
+  Working with a company that needs Stoplight? Provide the relavent information about your deal and our team will be in touch.
```
Comment (paulatulis) [inline netlify/forms/partner-deal.yaml]:
```suggestion
  Working with a company that needs Stoplight? Provide the relevant information about your deal and our team will be in touch.
```

File: netlify/forms/partner-deal.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+hero:
+  particles: true
+color: black
+hubspot:
+  formId: dc28e0a6-fe9c-4d7e-b6f6-966365a27577
```
Comment (paulatulis) [inline netlify/forms/partner-deal.yaml]: I don't work in this realm, so maybe this is obvious, but what does Deal Name mean? Will that be obvious to people landing on this page?

Comment (ksatx) [reply netlify/forms/partner-deal.yaml]: Yeah it should be

**Pull request merged**